features/git-prune-branches/feature_branch/behind_main/with_open_changes.feature

```diff
@@ -6,27 +6,49 @@ Feature: git prune-branches: don't remove the current empty feature branch if th
 
 
   Background:
-    Given I have a feature branch named "feature" behind main
-    And I have a feature branch named "stale_feature" behind main
-    And I am on the "feature" branch
+    Given the following commits exist in my repository
+      | BRANCH | LOCATION         | MESSAGE     | FILE NAME |
+      | main   | local and remote | main commit | main_file |
+    And I have a stale feature branch named "stale_feature_initial" with its tip at "Initial commit"
+    And I have a stale feature branch named "stale_feature_main" with its tip at "main commit"
```

Comment:
The reason why I do it the currently implemented way is that it allows me to verify on line 44 and 46 that each branch is correctly re-instantiated at their own respective tip. If both branches were cut directly from the same commit, the undo script could reinstantiate them in wrong ways, and we wouldn't know.